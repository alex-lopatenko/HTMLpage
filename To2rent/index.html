<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>to2rent Дома и квартиры</title>
  <!-- Yandex.Metrika counter -->
  <script type="text/javascript" >
    (function(m,e,t,r,i,k,a){m[i]=m[i]||function(){(m[i].a=m[i].a||[]).push(arguments)};
      m[i].l=1*new Date();k=e.createElement(t),a=e.getElementsByTagName(t)[0],k.async=1,k.src=r,a.parentNode.insertBefore(k,a)})
    (window, document, "script", "https://mc.yandex.ru/metrika/tag.js", "ym");

    ym(65043838, "init", {
      clickmap:true,
      trackLinks:true,
      accurateTrackBounce:true,
      webvisor:true
    });
  </script>
  <noscript><div><img src="https://mc.yandex.ru/watch/65043838" style="position:absolute; left:-9999px;" alt="" /></div></noscript>
  <!-- /Yandex.Metrika counter -->
  <link href="https://fonts.googleapis.com/css2?family=PT+Serif&family=Roboto+Condensed&display=swap&subset=cyrillic" rel="stylesheet">
  <link rel="stylesheet" href="css/reset.css">
  <link rel="stylesheet" href="css/slick.css">
  <link rel="stylesheet" href="css/style.css">
</head>
<body>
<header class="header">
  <div class="logo header__logo">
    <a class="logo__link" href="#">
      <img class="logo__link-img" src="images/logo.svg" alt="Логотип">
    </a>
  </div>
  <nav class="menu">
    <ul class="menu__list">
      <li class="menu__list-item">
        <a href="#" class="menu__list-link menu__list-link--active">О нас</a></li>
      <li class="menu__list-item">
        <a href="#" class="menu__list-link">Дома</a></li>
      <li class="menu__list-item">
        <a href="#" class="menu__list-link">Квартиры</a></li>
      <li class="menu__list-item">
        <a href="#" class="menu__list-link">Контакты</a></li>
    </ul>
  </nav>
</header>
<section class="slider">
  <div class="slider__item" style="background-image: url(images/slider-item.jpg)"></div>
  <div class="slider__item" style="background-image: url(images/slider-item.jpg)"></div>
  <div class="slider__item" style="background-image: url(images/slider-item.jpg)"></div>
</section>
<section class="search">
    <div class="search__inner">
      <form class="search__form">
        <label class="search__form-label">Поиск Апартаментов</label>
        <input type="text" class="search__form-input" placeholder="Введите город">
        <button class="search__form-btn" type="submit">Найти</button>
      </form>
    </div>
</section>
<section class="popular">
  <h3 class="popular__title">Самые популярные варианты</h3>
  <div class="container">
    <div class="popular__inner">

      <div class="popular__item">
        <img src="images/popular-1.jpg" alt="" class="popular__img">
        <h5 class="popular__item-title">Апартаменты <br> Москва</h5>
        <a href="" class="popular__link">детали</a>
      </div>

      <div class="popular__item">
        <img src="images/popular-2.jpg" alt="" class="popular__img">
        <h5 class="popular__item-title">Апартаменты <br> Киев</h5>
        <a href="" class="popular__link">детали</a>
      </div>

      <div class="popular__item">
        <img src="images/popular-3.jpg" alt="" class="popular__img">
        <h5 class="popular__item-title">Коттедж Modern <br> Лос Анджелес</h5>
        <a href="" class="popular__link">детали</a>
      </div>

      <div class="popular__item">
        <img src="images/popular-4.jpg" alt="" class="popular__img">
        <h5 class="popular__item-title">Коттедж Семейный <br> Лондон</h5>
        <a href="" class="popular__link">детали</a>
      </div>
    </div>
  </div>
</section>
<section class="reason">
  <div class="container">
    <div class="reason__inner">

      <div class="reason__item">
        <h4 class="reason__title reason__title--icon-1">
          Мы экономим
          Ваше время и Ваши
          деньги</h4>
        <p class="reason__text">
          Вы можете легко подобрать себе самую лучшую
          недвижимость для аренды на нашем сайте по выгодной
          цене. Наши менеджеры с удовольствием помогут Вам и
          ответят на все вопросы.</p>
        <a href="#" class="reason__link">Подробнее</a>
      </div>

      <div class="reason__item">
        <h4 class="reason__title reason__title--icon-2">
          Арендуйте лучшую
          недвижимость в
          крупных городах
        </h4>
        <p class="reason__text">
          Наша база элитной недвижимости насчитывает более 5000
          вариантов в крупных городах мира.Мы поможем Вам
          выбрать самую лучшую, также предоставим дополнительный
          сервис
        </p>
        <a href="#" class="reason__link">Подробнее</a>
      </div>

    </div>
  </div>
</section>
<section class="best" style="background-image: url(images/best.jpg)">
  <div class="container">
    <h3 class="best__title">
      Коттедж Modern
      Лос Анджелес</h3>
    <a href="#" class="best__link">Забронировать</a>
  </div>
</section>
<section class="about">
  <div class="container">
    <div class="about__inner">
      <div class="our-managers">
        <h4 class="our-managers__title about__title">
          Наши менеджеры
        </h4>
        <div class="our-managers__items">

          <div class="our-managers__item">
            <img src="images/our-managers-1.jpg" class="our-managers__img" alt="">
            <ul class="our-managers__list">
              <li class="our-managers__name">Артур</li>
              <li class="our-managers__age">30 лет</li>
              <li class="our-managers__location">Москва</li>
            </ul>
            <button class="our-managers__btn">Связаться</button>
          </div>

          <div class="our-managers__item">
            <img src="images/our-managers-2.jpg" class="our-managers__img" alt="">
            <ul class="our-managers__list">
              <li class="our-managers__name">Джессика</li>
              <li class="our-managers__age">28 лет</li>
              <li class="our-managers__location">Лондон</li>
            </ul>
            <button class="our-managers__btn">Связаться</button>
          </div>
        </div>
      </div>

        <div class="blog">
          <h4 class="about__title">Блог</h4>
          <ul class="blog__list">
            <li class="blog__item">
              <a class="blog__link" href="#">
                Системы охраны и видеонаблюдения
              </a>
            </li>
            <li class="blog__item">
              <a class="blog__link" href="#">Клубная недвижимость всегда в цене</a>
            </li>
            <li class="blog__item">
              <a class="blog__link" href="#">Все плюсы от посуточной аренды квартир</a>
            </li>
            <li class="blog__item">
              <a class="blog__link" href="#">Как выбрать жильё своей мечты в Киеве?</a>
            </li>
            <li class="blog__item">
              <a class="blog__link" href="#">Топ 10 ВИП коттеджей Лос Анджелеса</a>
            </li>
            <li class="blog__item">
              <a class="blog__link" href="#">Лучшие аппартаменты с панорамным видом</a>
            </li>
            <li class="blog__item">
              <a class="blog__link" href="#">Дома для аренды на год в Майами</a>
            </li>
          </ul>
        </div>

        <div class="reviews">
          <h4 class="about__title">Отзывы</h4>
          <img src="images/reviews.jpg" alt="" class="reviews__img">
          <p class="reviews__text">
            Это лучший сервис по подбору
            аппартаментов. Пользуюсь услугами
            To rent уже 5 лет. Всегда идеально и
            именно то, что мне нужно.
          </p>
          <a href="#" class="reviews__link">Еще отзывы</a>
        </div>

      </div>
    </div>
  </div>
</section>

<footer class="footer">
  <div class="footer__top">
    <div class="container">
      <div class="footer__top-inner">
        <nav class="footer__menu">
          <ul class="footer__menu-list">
            <il class="footer__menu-item">
              <a href="#" class="footer__menu-link">
                О нас
              </a></il>
            <il class="footer__menu-item">
              <a href="#" class="footer__menu-link">
                Дома
              </a></il>
            <il class="footer__menu-item">
              <a href="#" class="footer__menu-link">
                Аппартаменты
              </a></il>
            <il class="footer__menu-item">
              <a href="#" class="footer__menu-link">
                Отзывы
              </a></il>
            <il class="footer__menu-item">
              <a href="#" class="footer__menu-link">
                Контакты
              </a></il>
          </ul>
        </nav>
        <form class="footer__form">
          <label class="footer__form-label">Связаться <br> с нами</label>
          <input class="footer__form-input" type="text" placeholder="Задать вопрос">
          <button class="footer__form-btn" type="submit">Отправить</button>
        </form>
      </div>
    </div>
  </div>
  <div class="logo footer__logo">
    <a class="logo__link" href="#">
      <img class="logo__link-img" src="images/logo.svg" alt="Логотип">
    </a>
  </div>
</footer>


  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
  <script src="js/slick.min.js"></script>
  <script src="js/main.js"></script>
</body>
</html>