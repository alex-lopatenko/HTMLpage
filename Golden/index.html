<!DOCTYPE html>
<html lang="en">
<head>
    <!-- Подключаемые файлы, метатеги, название страницы -->
    <meta charset="utf-8"> <!-- Кодировка страницы-->
    <title>Golden</title>
    <link rel="stylesheet" href="css/main.css">
    
    <!-- Yandex.Metrika counter -->
<script type="text/javascript" >
   (function(m,e,t,r,i,k,a){m[i]=m[i]||function(){(m[i].a=m[i].a||[]).push(arguments)};
   m[i].l=1*new Date();k=e.createElement(t),a=e.getElementsByTagName(t)[0],k.async=1,k.src=r,a.parentNode.insertBefore(k,a)})
   (window, document, "script", "https://mc.yandex.ru/metrika/tag.js", "ym");

   ym(62617549, "init", {
        clickmap:true,
        trackLinks:true,
        accurateTrackBounce:true,
        webvisor:true
   });
</script>
<noscript><div><img src="https://mc.yandex.ru/watch/62617549" style="position:absolute; left:-9999px;" alt="" /></div></noscript>
<!-- /Yandex.Metrika counter -->
</head>
<body>


<header>
    <div class="container">
        <div class="heading clearfix">
            <img src="img/logo.png" alt="Golden" class="logo">
            <nav>
                <ul class="menu">
                    <li>
                        <a href="#">Home</a>
                    </li>
                    <li>
                        <a href="#">Services</a>
                    </li>
                    <li>
                        <a href="#">Portfolio</a>
                    </li>
                    <li>
                        <a href="#">About</a>
                    </li>
                    <li>
                        <a href="#">Contact</a>
                    </li>
                </ul>
            </nav>
        </div>
        <div class="titles">
            <div class="titles__first">
                Welcome To Our Studio!
            </div>
            <h1>
                It’s nice to Meet you
            </h1>
        </div>
        <a class="button" href="#">Tell Me More</a>
    </div>
</header>


<section id="services">
    <div class="container">
        <div class="title">
            <h2>
                Services
            </h2>
            <p>
                Proin iaculis purus consequat sem cure.
            </p>
        </div>
        <div class="services clearfix">
            <div class="services__item">
                <img src="img/icon1.png" alt="Услуга">
                <h3>E-Commerce</h3>
                <p>
                    Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit
                    aenean rhoncus posuere odio in tincidunt.
                </p>
            </div>
            <div class="services__item">
                <img src="img/icon2.png" alt="Услуга">
                <h3>Responsive Web</h3>
                <p>
                    Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit
                    aenean rhoncus posuere odio in tincidunt.
                </p>
            </div>
            <div class="services__item">
                <img src="img/icon3.png" alt="Услуга">
                <h3>Web Security</h3>
                <p>
                    Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit
                    aenean rhoncus posuere odio in tincidunt.
                </p>
            </div>
        </div>
    </div>
</section>
<section id="portfolio">
    <div class="container">
        <div class="title">
            <h2>
                Our portfolio
            </h2>
            <p>
                Proin iaculis purus consequat sem cure.
            </p>
        </div>
        <div class="works clearfix">
            <img src="img/img1.jpg" alt="Работа">
            <img src="img/img2.jpg" alt="Работа">
            <img src="img/img3.jpg" alt="Работа">
        </div>
    </div>
</section>
<div id="about" name="about" class="box">
    <div class="container">
        <h2 class="title">About Us</h2>
        <p class="descr">Proin iaculis purus consequat sem cure.</p>
        <div class="about-wrap">
            <div class="about-item clearfix">
                <div class="about-text left text-right">
                    <h5 class="title">
                        July 2010<br/> Our Humble Beginnings
                    </h5>
                    <p>
                   Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit.
                    </p>
                </div>
                <div class="about-img left">
                    <img src="img/about1.jpg" alt="">
                </div>
            </div>
            <div class="about-item clearfix">
                <div class="about-text right text-left">
                    <h5 class="title">
                        January 2011<br/>
                        Facing Startup Battles
                    </h5>
                    <p>
                   Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit aenean 
rhoncus posuere odio in tincidunt.
                    </p>
                </div>
                <div class="about-img_1 right">
                    <img src="img/about2.jpg" alt="">
                </div>
            </div>
            <div class="about-item clearfix">
                <div class="about-text left text-right">
                    <h5 class="title">
                        December 2012<br/>Enter The Dark Days
                    </h5>
                    <p>
                   Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni.
                    </p>
                </div>
                <div class="about-img left">
                    <img src="img/about3.jpg" alt="">
                </div>
            </div>
            <div class="about-item clearfix">
                <div class="about-text right text-left">
                    <h5 class="title">
                        february 2014<br/>
                        Our Triumph
                    </h5>
                    <p>
                   Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit aenean.
                    </p>
                </div>
                <div class="about-img_1 right">
                    <img src="img/about4.jpg" alt="">
                </div>
            </div>
        </div>
    </div>
</div>
<footer>
    <div class="container">
       <div class="footer__copyright">
           © Copyright 2014 FreebiesXpress.com
       </div> 
    </div>

</footer>
</body>
</html>
